.coafile

```diff
@@ -9,7 +9,7 @@ bears = SpaceConsistencyBear
 default_actions = *: ApplyPatchAction
 
 [all.linelength]
-ignore += **.html
+ignore += **.html, static/js/i18n/**.json
```

Comment:
ugh?  Why is this?

They should all be unicode, and should all be nicely structured JSON.

Reply:
JSON doesn't have linebreaks and the translations for some strings are too long for the linelength limit.